src/kixi/heimdall/util.clj

```diff
@@ -25,3 +26,8 @@
   "Convers hyphens to underscores"
   [x]
   (replacer #(clojure.string/replace % #"-" "_") x))
+
+(defn file-exists?
+  [& path]
+  (.exists (apply io/file path)))
```

Comment:
I dislike such things (non-clojure bias) - i think both types returned should be same :)